pkg/featureflag/feature_flag.go

```diff
@@ -30,6 +34,8 @@ func InitFeatureFlag(ctx context.Context) *FlagIdentifier {
 	var featureFlags FlagIdentifier
 	featureFlags.enabledFeatures = enabledFeatures
 
+	log.G(ctx).Infof("features %v enabled", enabledFeatures)
```

Comment:
@helayoty let me know what you think about this fix. The logging was a bit verbose: each time checking for a feature would log something.

Reply:
We can update it to be in a Debug level.